README.md

```diff
@@ -76,6 +76,8 @@ API documentation can be found in [API.md](./API.md).
 
 ## Build
 
+**IMPORTANT: As documented under [Build Dependencies](#build-dependencies), the minimum version requirement for the iRODS development package is 4.3.2 or later. As a result, support for the GenQuery2 API plugin has been removed. To use GenQuery2, the iRODS HTTP API must be connected to a server running iRODS 4.3.2 or later.**
```

Comment:
What happens if the HTTP API is connected to an iRODS server running 4.3.1 or earlier? It falls back to GenQuery1?

Reply:
I was a little confused by the "To use GenQuery2" clause because my understanding is that the HTTP API is going to use GenQuery2 no matter what, but now it's going to use the one baked into the server which is only available in iRODS 4.3.2 and later, and therefore will not work with iRODS 4.3.1 and earlier.

If this is the case, I think this should say...

> The iRODS HTTP API must be connected to a server running iRODS 4.3.2 or later because the HTTP API now uses the GenQuery2 API which is built into the server rather than the GenQuery2 API plugin.

Instead of...

> To use GenQuery2, the iRODS HTTP API must be connected to a server running iRODS 4.3.2 or later.

Thoughts?

Reply:
I thought it knew which server version it was attached to... and behaved accordingly...?

Reply:
Users can still run the HTTP API against 4.3.1. They just won't be able to invoke GenQuery2 _(even though the HTTP API has support for it built in due to it being compiled against 4.3.2 dev libs)_.

The HTTP API still defaults to GenQuery1 for the /query endpoint. If a user wants to use/call GenQuery2, they must set the appropriate parameter in the HTTP request and the HTTP API must be connected to a 4.3.2 (or later) server.

Only one implementation of GenQuery2 is supported because the GenQuery2 API plugin repo does not compile against 4.3.2. _Of course this could be made to work, but it didn't seem worth the effort at the end of the day._

Reply:
Ah, okay, that makes sense. Yes, resolving.